packages/wallet_spec/wallet_spec.md

```diff
@@ -0,0 +1,31 @@
+## What is the role of the magmo channel wallet?
```

Comment:
What's the purpose of the `wallet_spec` package?

Reply:
It shouldn't be a package -- just some rough notes that don't currently have a good home. Will remove.

Reply:
Currently found under `/wallet/notes/'